conmon/conmon.c

```diff
@@ -740,9 +790,19 @@ static bool read_stdio(int fd, stdpipe_t pipe, gboolean *eof)
 		nwarnf("stdio_input read failed %s", strerror(errno));
 		return false;
 	} else {
-		if (write_k8s_log(log_fd, pipe, buf, num_read) < 0) {
-			nwarn("write_k8s_log failed");
-			return G_SOURCE_CONTINUE;
+		if (g_log_driver == JOURNALD) {
```

Comment:
here we can use a `switch (g_log_driver)` where the `default` case exits with an error